### PR TITLE
chore: force to stop pipeline when next request in

### PR DESCRIPTION
**Pull request by Decodetalkers**
sometimes the pipeline will be blocked, so here mv the pipeline read to QtCurrent runtime, when next request is in , close the pipeline

Log:

### CODE REVIEW

**Comment by justforlxz** [comment]
/+1

File: dde-clipboard-daemon/readpipedatatask.h [hunk 1]
```diff
@@ -25,23 +23,24 @@ class ReadPipeDataTask : public QObject
...
+                              const QString &mimeType, QObject *parent = nullptr);
 
-    void stopRunning();
     void run();
 
 signals:
     void dataReady(qint64, const QString &, const QByteArray &);
-
-
+    // In some applications like chrome, when it sends data, seems kwin cannot handle the request good enough, sometimes also firefox
```
Comment (BLumia) [inline dde-clipboard-daemon/readpipedatatask.h:32]: 有具体的例子/情况说明吗？这个描述有点模糊，想了解下具体情况。

**Review by BLumia — CHANGES_REQUESTED** [review]
日常批判 typo：

- `QtCurrent` > `QtConcurrent`
- `when next request is in ,` > `when next request is in,` （而且这个描述本身也有点 chinglish）

**Review by BLumia — APPROVED** [review]

**Pull request merged**